apps/frontend/src/pages/quiz/libs/components/balance-wheel/balance-wheel.tsx

```diff
@@ -17,12 +20,18 @@ const BalanceWheel: React.FC = () => {
 				return previousPercentage;
 			}
 
-			const newPercentage =
-				previousPercentage + PercentageConfig.INCREMENT_VALUE;
+			const nextIndex = PercentageConfig.PERCENTAGE_VALUES.findIndex(
```

Comment:
Use INCREMENT_VALUE, change it to 12.5 and display rounded number. The percentages should be 13, 25, 38, 50, 63, 75, 88, 100